### PR TITLE
[MIG] crm_lead_firstname: Migration to 16.0

### PR DESCRIPTION
**Pull request by XanderDeMatteis**
Includes https://github.com/OCA/crm/pull/456

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -113,7 +113,7 @@ repos:
           - requirements.txt
           - --header
           - "# generated from manifests external_dependencies"
-  - repo: https://gitlab.com/PyCQA/flake8
+  - repo: https://github.com/PyCQA/flake8
```
Comment (StefanRijnhart) [inline .pre-commit-config.yaml]: Fixed now in https://github.com/OCA/crm/pull/462. Please rebase.

**Review by StefanRijnhart — APPROVED** [review]
Thanks! Code reviewed + tested on runboat.

**Review by rven — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**